model_server/pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "stampy_nlp_model"
+dynamic = ["version"]
+authors = [
+    {name = "ccstan99", email = "hello@stampy.ai"},
+    {name = "Daniel O'Connell", email = "github@ahiru.pl"},
+]
+description = "A simple server to encapsulate LLMs as microservices"
+readme = "README.md"
+requires-python = ">=3.8"
+license = {text = "MIT"}
+dependencies = [
+    "starlette",
+    "asyncio",
```

Comment:
fastapi uses starlette under the hood. I went with starlette because it can handle async a lot better than flask can. It should be totally fine for only this to use starlette, as it's supposed to be a very simple service.
I'm still somewhat suspicious of flask. It's grown and matured a lot over the years, but it started out as an [april fools joke](https://lucumr.pocoo.org/2010/4/3/april-1st-post-mortem/). I don't want to bash it though - it's a lot nicer than say Django

Reply:
Interesting! Thanks for sharing.